packages/sanes-chrome-extension/src/context/PersonaProvider.tsx

```diff
@@ -22,35 +22,37 @@ export const PersonaContext = React.createContext<PersonaContextInterface>({
 
 interface Props {
   readonly children: React.ReactNode;
+  readonly persona: GetPersonaResponse;
 }
 
-export const PersonaProvider = ({ children }: Props): JSX.Element => {
-  const [accountNames, setAccountNames] = React.useState<Accounts>([]);
-  const [mnemonic, setMnemonic] = React.useState<string>('');
-  const [txs, setTxs] = React.useState<ReadonlyArray<ProcessedTx>>([]);
-  const isExtensionContext = typeof chrome !== 'undefined';
-
+export const PersonaProvider = ({ children, persona }: Props): JSX.Element => {
```

Comment:
@webmaster128 @abefernan @albertandrejev now passing `persona: GetPersonaResponse` here for hydrating data, will solve much time doing DOM testing because we can mock it 🎉